### PR TITLE
Issue #12077: Github action to update sources with release notes

### LINKED ISSUE #12077: Github action to update sources with release notes
https://github.com/checkstyle/contribution/blob/master/releasenotes-builder/diagrams-manual-trigger.md

we need to make action to do `run Github action "Push releasenotes '10.3.1'"`  (action takes human provided version as input, just to make sure it can not be executed unintentionally)
if user provider input is not matching pom - fail build
so we simply make script that update releasenotes.xml
and github action commit and push changes

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Fixes #12077

### CODE REVIEW

File: .ci/update-sources.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ./.ci/util.sh
+
+if [[ -z $1 ]]; then
+  echo "latest version is not set"
```
Comment (romani) [inline .ci/update-sources.sh:8]: remove "latest"

File: .github/workflows/update_sources.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#############################################################################
+# GitHub Action to Push Release Notes.
+#
+#############################################################################
+name: "Push Release Notes"
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'New Version'
```
Comment (romani) [inline .github/workflows/update_sources.yml:10]: `Target Version without '-SNAPSHOT'`

**Review by romani — CHANGES_REQUESTED** [review]
item:

File: .ci/update-sources.sh (hunk 1 above)
Comment (Rahulkhinchi03) [reply .ci/update-sources.sh]: Changes Done

File: .github/workflows/update_sources.yml (hunk 2 above)
Comment (Rahulkhinchi03) [reply .github/workflows/update_sources.yml]: Changes Done

File: .ci/update-sources.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ./.ci/util.sh
+
+if [[ -z $1 ]]; then
+  echo "version is not set"
+  echo "Usage: outside-sources.sh <version>"
```
Comment (romani) [inline .ci/update-sources.sh:9]: Let's make name of file
`update-releasenotesxml.sh`

Comment (romani) [inline .ci/update-sources.sh:9]: Let's make name of file
`update-releasenotesxml.sh`

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: .ci/update-sources.sh (hunk 3 above)
Comment (Rahulkhinchi03) [reply .ci/update-sources.sh]: Done

Comment (Rahulkhinchi03) [reply .ci/update-sources.sh]: Done

**Review by romani — APPROVED** [review]
ok to merge

**Pull request merged**